test/exports.test.js

```diff
@@ -1,5 +1,5 @@
 import assert from 'assert';
-import validator from '../index';
```

Comment:
FYI: I don't know what this file is used for, but `npm run lint:fix` errors here.